packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -385,19 +383,22 @@ export class AggregatedAttestationPool {
             if (score < minScore) {
               minScore = score;
             }
-            consolidations.push({...consolidation, score});
+
+            consolidations.set(consolidation, score);
+
             // Stop accumulating attestations there are enough that may have good scoring
-            if (consolidations.length >= MAX_ATTESTATIONS_ELECTRA * 2) {
+            if (consolidations.size >= MAX_ATTESTATIONS_ELECTRA * 2) {
               break slot;
             }
           }
         }
       }
     }
-
-    const sortedConsolidationsByScore = consolidations
-      .sort((a, b) => b.score - a.score)
+    const sortedConsolidationsByScore = [...consolidations.entries()]
+      .sort((a, b) => b[1] - a[1])
+      .map(([consolidation, _]) => consolidation)
       .slice(0, MAX_ATTESTATIONS_ELECTRA);
+
```

Comment:
Noticed are not cloning here
https://github.com/ChainSafe/lodestar/blob/e45e0ebd4be70e91cdaa028d00e823b9621a7943/packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts#L288-L289

but might be fine due to `aggregateConsolidation`